counterexamples/divisions/division_boundary/missing-country.yaml

```
@@ -8,10 +8,10 @@ properties:
   theme: divisions
   type: division_boundary
   version: 0
-  subtype: region
+  subtype: admin1
   is_territorial: true
   is_land: false
   class: land
-  division_ids: ["example:division:country:left", "example:division:country:right"]
+  division_ids: ["example:division:admin0:left", "example:division:admin0:right"]
```

Seems like nobody cared about these ids to begin with :D
```suggestion
  division_ids: ["example:division:admin1:left", "example:division:admin1:right"]
```